_gcode/M605.md

```diff
@@ -26,6 +26,9 @@ parameters:
       -
         tag: 2
         description: Duplication mode. Carriages and extruders move in unison.
+	  - 
+	    tag: 3
+		description: Mirrored mode. Carriages extruder duplicates the movement of the first except the second extruder is reversed in the X axis.
```

Comment:
Use spaces instead of tabs & watch for alignment of each `tag:`

Reply:
ok, i will ajust too

Reply:
Requested adjustments made and committed.